sda/cmd/verify/verify.go

```diff
@@ -41,8 +41,8 @@ func main() {
 		log.Fatal(err)
 	}
 	archiveKeyList, err := config.GetC4GHprivateKeys()
-	if err != nil {
-		log.Fatal(err)
+	if err != nil || len(archiveKeyList) == 0 {
+		log.Fatal("no C4GH private keys configured")
```

Comment:
Would be nice to provide error if this is because of one?

Reply:
I agree. Maybe something like this:

```suggestion
if err != nil {
	log.Fatalf("encountered an error: %v", err)
}
if len(archiveKeyList) == 0 {
	log.Fatal("no C4GH private keys configured")
}
```

Reply:
The errors are printed in the`GetC4GHprivateKeys` function